integrated_channels/moodle/client.py

```diff
@@ -116,3 +136,31 @@ def delete_content_metadata(self, serialized_data):
             params=params
         )
         return response.status_code, response.text
+
+    def _get_access_token(self):
+        querystring = {
+            'service': self.enterprise_configuration.service_short_name
```

Comment:
Is this only required for token requests? Mostly just curious since I have never included it, but since tokens seem to be assigned per webservice that might make sense.

Reply:
Yup! This'll give you the token for a service with specific functions/permissions. That token is then used in Moodle elsewhere (and I'm guessing they resolve the token to a service to get perms) so you don't need to specify that name in other requests.